File: pgm/ABC_Java/ABC_216_A.java
import java.io.*;
// import java.math.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final String dot = "\\.";
        final String[] arr = reader.readLine().split (dot);
        final String minus = "-";
        final String plus = "+";
        final int two = 2;
        final int six = 6;
        int X = Integer.parseInt (arr[0]);
        final int Y = Integer.parseInt (arr[1]);
        if (Y <= two) {
            System.out.print (X);
            System.out.print (minus);
        } else if (Y <= six) {
            System.out.print (X);
        } else {
            System.out.print (X);
            System.out.print (plus);
        }
        System.out.println();
        return;
    }
}

File: pgm/ABC_Kotlin/ABC_216_A.kt
// import java.io.BufferedOutputStream
// import java.io.BufferedReader
// import java.io.DataInputStream
// import java.io.FileInputStream
// import java.io.IOException
// import java.io.InputStreamReader
// import java.io.PrintWriter
// import java.lang.StringBuilder
// import java.time.LocalTime
// import java.time.format.DateTimeFormatter
// import java.util.Scanner
// import java.util.StringTokenizer

// import kotlin.math.*

const val minus: String = "-"
const val dot: String = "."
const val plus: String = "+"
const val blank: String = ""
const val two: Int = 2
const val six: Int = 6

fun main() {
    val (X: Int, Y: Int) = readln().split(dot).map { it.toInt() }
    val sign: String =
            when {
                Y <= two -> minus
                Y <= six -> blank
                else -> plus
            }
    println("$X$sign")
    kotlin.system.exitProcess(0)
}